script/vsts/platforms/macos.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       - script: script/bootstrap
         displayName: Bootstrap build environment
         env:
+          GITHUB_TOKEN: $(GITHUB_TOKEN)
```

Comment:
You should do it for all operating systems.

Reply:
Done!